<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>701</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>480</width>
      <height>640</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>视频检测区域</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="MyButton" name="btn_cap" native="true">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>380</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>摄像头</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>390</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>检测</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="ChartShow" name="widget_chart" native="true">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>631</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="MyButton" name="btn_dete" native="true">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>380</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>480</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_cputext">
     <property name="geometry">
      <rect>
       <x>3</x>
       <y>10</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#3465a4;&quot;&gt;CPU&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_cpu">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>温度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>43</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#f57900;&quot;&gt;℃&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(114, 159, 207);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="hs_speed">
    <property name="geometry">
     <rect>
      <x>829</x>
      <y>481</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { border: 0px solid #bbb; } 
QSlider::sub-page:horizontal { background: rgb(235,97,0); border-radius: 0px; margin-top:8px; margin-bottom:8px; } 
QSlider::add-page:horizontal { background: rgb(200,200, 200); border: 0px solid #777; border-radius: 2px; margin-top:8px; margin-bottom:8px; } 
QSlider::handle:horizontal { background: rgb(255,153,102); border: 1px solid rgb(255,153,102); width: 25px; height:4px; border-radius: 7px; margin-top:2px; margin-bottom:2px; } 
QSlider::handle:horizontal:hover { background: rgb(255,128,6); border: 1px solid rgba(102,102,102,102); border-radius: 7px; } 
QSlider::sub-page:horizontal:disabled { background: #bbb; border-color: #999; } 
QSlider::add-page:horizontal:disabled { background: #eee; border-color: #999; } 
QSlider::handle:horizontal:disabled { background: #eee; border: 1px solid #aaa; border-radius: 4px; }</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>493</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>转速</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>493</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>554</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>高度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_height">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>553</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="hs_height">
    <property name="geometry">
     <rect>
      <x>829</x>
      <y>540</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { border: 0px solid #bbb; } 
QSlider::sub-page:horizontal { background: rgb(235,97,0); border-radius: 0px; margin-top:8px; margin-bottom:8px; } 
QSlider::add-page:horizontal { background: rgb(200,200, 200); border: 0px solid #777; border-radius: 2px; margin-top:8px; margin-bottom:8px; } 
QSlider::handle:horizontal { background: rgb(255,153,102); border: 1px solid rgb(255,153,102); width: 25px; height:4px; border-radius: 7px; margin-top:2px; margin-bottom:2px; } 
QSlider::handle:horizontal:hover { background: rgb(255,128,6); border: 1px solid rgba(102,102,102,102); border-radius: 7px; } 
QSlider::sub-page:horizontal:disabled { background: #bbb; border-color: #999; } 
QSlider::add-page:horizontal:disabled { background: #eee; border-color: #999; } 
QSlider::handle:horizontal:disabled { background: #eee; border: 1px solid #aaa; border-radius: 4px; }</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="hs_foreward">
    <property name="geometry">
     <rect>
      <x>829</x>
      <y>602</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal { border: 0px solid #bbb; } 
QSlider::sub-page:horizontal { background: rgb(235,97,0); border-radius: 0px; margin-top:8px; margin-bottom:8px; } 
QSlider::add-page:horizontal { background: rgb(200,200, 200); border: 0px solid #777; border-radius: 2px; margin-top:8px; margin-bottom:8px; } 
QSlider::handle:horizontal { background: rgb(255,153,102); border: 1px solid rgb(255,153,102); width: 25px; height:4px; border-radius: 7px; margin-top:2px; margin-bottom:2px; } 
QSlider::handle:horizontal:hover { background: rgb(255,128,6); border: 1px solid rgba(102,102,102,102); border-radius: 7px; } 
QSlider::sub-page:horizontal:disabled { background: #bbb; border-color: #999; } 
QSlider::add-page:horizontal:disabled { background: #eee; border-color: #999; } 
QSlider::handle:horizontal:disabled { background: #eee; border: 1px solid #aaa; border-radius: 4px; }</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>614</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>前进</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_foreward">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>614</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>659</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/picture/exit.png</normaloff>:/picture/exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_ip">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>660</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备已连接</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_wifi_connected">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>650</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/picture/connected.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_test">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>660</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>状态:   正在运行</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QWidget</extends>
   <header>mybutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChartShow</class>
   <extends>QWidget</extends>
   <header>chartshow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
